### PR TITLE
#1331 feedback link

### PR DESCRIPTION
**Pull request by vandnakapoor19**
Before submitting a final PR, please:

- [ ] Run `npm test` on your machine
- [ ] Run `npm run format` to keep code formatted consistently
- [ ] Reference the issue you are fixing in at least one of your commit messages, as `Fixes #X`

### CODE REVIEW

File: extension/popup/popup.css [hunk 1]
```diff
@@ -119,12 +119,11 @@ body,
...
 #moz-voice-privacy {
   line-height: 20px;
-  margin-right: 32px;
+  margin-left: -16px;
 }
-
 #moz-voice-privacy > a {
   font-weight: 400;
-  font-size: 0.85em;
+  font-size: 0.95em;
```
Comment (ianb) [inline extension/popup/popup.css]: I'm guessing that only this popup.css change is intended to be included here?

There's also changes related to music players.

Comment (vandnakapoor19) [reply extension/popup/popup.css]: @ianb Yes, I've worked on only popup.css file.

Comment (vandnakapoor19) [reply extension/popup/popup.css]: @ianb and also, I am not sure why these all other file changes are reflecting in my pull request. I have only made two changes to my popup.css file for this issue.

**Comment by ianb** [comment]
I think you got this added in 471d35e1

**Pull request closed without merging**